test/users.js

```diff
@@ -728,3 +730,576 @@ describe('/users GET', () => {
       .catch(err => done(err));
   });
 });
+
+describe('/users/admin POST', () => {
+  it('should save the new user in the database with role "admin" when requester is admin', done => {
+    const validToken = jwt.encode(
+      { id: 3, email: 'noctis.lucis@wolox.com.ar' },
+      config.common.session.secret
+    );
+
+    const validUser = {
+      firstName: 'John',
+      lastName: 'Doe',
+      email: 'john.doe@wolox.cl',
+      password: '12345678'
+    };
+    chai
+      .request(server)
+      .post('/users/admin')
+      .set(config.common.session.header_name, validToken)
+      .send(validUser)
+      .then(res => {
+        res.should.have.status(201);
+        res.should.be.json;
+        expect(res.body.password).to.be.undefined;
+        expect(res.body.role).to.equal('admin');
+
+        dictum.chai(res, 'The saved user with role admin');
+        done();
+      })
+      .catch(err => done(err));
+  });
+
+  it('should update the user if it is already registered and update only the role', done => {
+    const validToken = jwt.encode(
+      { id: 3, email: 'noctis.lucis@wolox.com.ar' },
+      config.common.session.secret
+    );
+
+    const validUserId = 1;
+    const validUser = {
+      firstName: 'BLAHBLAHBLAH',
```

Comment:
this is just for you to know that i read this :)

Reply:
Awesome! But, is there something wrong with that string?

Reply:
No!